specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4366,13 +4366,27 @@
       "description": "settings for security policy patterns to match",
       "type": "object",
       "properties": {
+        "IsProfileLevel": {
```

Comment:
Why not make it a profile property? e.g. profile.properties.securityPolicy.
If we make it SecurityPolicy property, there will be implict restrictions between differnt SecurityPolicies and there will be atomic operation issues when you change profile level securityPolicy from policy1 to policy2

Reply:
Good point, but we don't have any references to security policies currently. All references are implemented in the other direction (security policy to sub-resource).

Reply:
We have decided to remove this flag from the Security Policy properties. In the PartnerData bundle file, we add a new key value pair "PolicyByProfile": WAF policy to indicate the presence of a profile level WAF.